_pytest/doctest.py

```diff
@@ -291,6 +295,18 @@ def _get_allow_bytes_flag():
     return doctest.register_optionflag('ALLOW_BYTES')
 
 
+def _get_report_choices():
+    import doctest
+    return dict(
+        zip(
+            _get_report_choices_keys(),
+            (doctest.REPORT_UDIFF, doctest.REPORT_CDIFF, doctest.REPORT_NDIFF, doctest.REPORT_ONLY_FIRST_FAILURE, 0, )
+        )
+    )
+
+def _get_report_choices_keys():
```

Comment:
Oh I see. OK that makes sense, thanks!

Could you add a little explanation to `get_report_choices_keys()` like "return report choices keys. separate function because this is used to declare options and we can't import doctest at that time"? Just so the next person who stumbles on this doesn't wonder the same thing. :grin: